rfc6962-bis.xml

```diff
@@ -464,6 +471,9 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
         <t>
           Note that the ASN.1 length and the opaque vector length are identical in size (1 byte) and value, so the DER encoding of the OID can be reproduced simply by prepending an OBJECT IDENTIFIER tag (0x06) to the opaque vector length and contents.
         </t>
+        <t>
+          OIDs used to identify logs are limited such that their DER encoding is less than 127 octets.
```

Comment:
"less than 127" isn't quite correct.

"less than 130" would be correct: 1 byte for the OBJECT IDENTIFIER tag, 1 byte for the length (with value 0x7F), and a 127-byte long value.

Perhaps change it to "...such that the DER encodings of their values are each <= 127 octets" ?
Or else express it in the singular (i.e. "An OID used to identify a log is limited such that the DER encoding of its value is <= 127 octets")?